pinot-core/src/main/java/org/apache/pinot/core/operator/dociditerators/ExpressionScanDocIdIterator.java

```diff
@@ -315,6 +326,10 @@ private void processProjectionBlock(ProjectionBlock projectionBlock, BitmapDataP
       }
     } else {
       // TODO(https://github.com/apache/pinot/issues/10882): support NULL for multi-value.
+      if (_nullHandlingEnabled || _predicateEvaluationResult == PredicateEvaluationResult.NULL) {
+        throw new UnsupportedOperationException("NULL handling is not supported for multi-value");
```

Comment:
We don't want to throw exception here. Think of a case where user needs null handling for other operators, but has a filter on MV column